spec/services/document_unpublishing_service_spec.rb

```diff
@@ -29,11 +33,30 @@
 
       assert_not_requested asset_manager_delete_asset(asset.asset_manager_id)
     end
+
+    it "adds an entry in the timeline of the document" do
+      stub_publishing_api_unpublish(document.content_id, body: { type: "withdrawal", explanation: explanatory_note, locale: document.locale })
+      DocumentUnpublishingService.new.retire(document, explanatory_note)
+
+      expect(document.timeline_entries.first.entry_type).to eq("retired")
+      expect(document.timeline_entries.first.retire.explanatory_note).to eq(explanatory_note)
+    end
+
+    it "keeps track of the live state" do
+      stub_publishing_api_unpublish(document.content_id, body: { type: "withdrawal", explanation: explanatory_note, locale: document.locale })
+      DocumentUnpublishingService.new.retire(document, explanatory_note)
+
+      expect(document).to have_received(:update!).with(live_state: "retired")
```

Comment:
Could we check the document directly here? I think we stubbed update! elsewhere only when we needed to inspect the state halfway through the method - should be OK here :-).

Reply:
(You can always use .reload to be sure we actually saved something.)

Reply:
I think I was following the same pattern in another test file for consistency.  But, yeah, I can change to actually check the value of `live_state` on the document.